contracts/test/PlonkVerifier.t.sol

```diff
@@ -493,24 +303,20 @@ contract PlonkVerifier_validateProof_Test is PlonkVerifierCommonTest {
 
 contract PlonkVerifier_preparePcsInfo_Test is PlonkVerifierCommonTest {
     /// @dev Test `preparePcsInfo` matches that of Jellyfish
-    function testFuzz_preparePcsInfo_matches(
-        uint64 seed,
-        uint256[] memory publicInput,
-        bytes memory extraTranscriptInitMsg
-    ) external {
+    function testFuzz_preparePcsInfo_matches(uint64 seed, uint256[] memory publicInput) external {
         publicInput = sanitizeScalarFields(publicInput);
         IPlonkVerifier.VerifyingKey memory vk = sanitizeVk(VkTest.getVk(), publicInput.length);
         IPlonkVerifier.PlonkProof memory proof = dummyProof(seed);
 
-        V.PcsInfo memory info = V._preparePcsInfo(vk, publicInput, proof, extraTranscriptInitMsg);
+        V.PcsInfo memory info = V._preparePcsInfo(vk, publicInput, proof);
 
         string[] memory cmds = new string[](6);
         cmds[0] = "diff-test";
         cmds[1] = "plonk-prepare-pcs-info";
         cmds[2] = vm.toString(abi.encode(vk));
         cmds[3] = vm.toString(abi.encode(publicInput));
         cmds[4] = vm.toString(abi.encode(proof));
-        cmds[5] = vm.toString(abi.encode(extraTranscriptInitMsg));
+        cmds[5] = vm.toString(abi.encode(""));
 
```

Comment:
can we update `diff_test/main.rs` to remove this argument?

Reply:
Sure, done in 2974abcd.